src/components/DateInput/index.js

```diff
@@ -6,43 +6,43 @@ export class DateInput extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            start: "2002-05-06",
-            end: Moment().format('YYYY-MM-DD'),
+            startDate: "2002-05-06",
+            endDate: Moment().format('YYYY-MM-DD'),
         }
     }
 
-    startChange = (e) => {
+    startDateChange = (e) => {
         this.setState({
-            start: e.target.value
+            startDate: e.target.value
         });
     }
-    endChange = (e) => {
+    endDateChange = (e) => {
         this.setState({
-            end: e.target.value
+            endDate: e.target.value
         });
     }
     componentDidMount(){
-        this.props.getLaunches(this.state.start, this.state.end);
+        this.props.getLaunches(this.state.startDate, this.state.endDate);
```

Comment:
A common convention is to destructure when you have multiple keys off of an object. So you might consider:

```
const { startDate, endDate } = this.state;
this.props.getLaunches(startDate, endDate);
```

That way you're not repeating `this.state`.